indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/PartialSegmentMergeTask.java

```diff
@@ -294,7 +300,20 @@ private Set<DataSegment> mergeAndPushSegments(
             exception -> !(exception instanceof NullPointerException) && exception instanceof Exception,
             5
         );
+        long pushFinishTime = System.nanoTime();
         pushedSegments.add(segment);
+        LOG.info(
+            "Segment[%s] of %,d bytes "
+            + "built from %d partial segment(s) in %,dms; "
+            + "pushed to deep storage in %,dms. "
+            + "Load spec is: %s",
+            segment.getId(),
+            segment.getSize(),
+            segmentFilesToMerge.size(),
+            (mergeFinishTime - startTime) / 1000000,
+            (pushFinishTime - mergeFinishTime) / 1000000,
+            jsonMapper.writeValueAsString(segment.getLoadSpec())
```

Comment:
Why do we want to log the loadspec ?

Reply:
Should we also log the interval ?

Reply:
The load is logged at all other places, so this keeps the parity about the deep storage segment info.
The interval is logged as a part of `SegmentId`

Reply:
I don't think it's worth passing in an `ObjectMapper` for it.  `.toString()` is hopefully good enough.  So, please remove the `ObjectMapper` from the constructor and just toString the thing.

Also, this log message can be better.  You can see a write-up of the idea here: https://github.com/apache/druid/blob/master/dev/style-conventions.md

Perhaps something like 

`"built segment [%s] ([%,d] input segments in [%,d] ms) and pushed ([%,d] ms) to deep storage [%s]"`